### PR TITLE
♻️ Updated Tooltip tokens

### LINKED ISSUE #1252: Conform Tooltip tokens

### PR DESCRIPTION
**Pull request by mimarz**
resolves #1252 

before:
image

after:
image

### CODE REVIEW

File: libraries/core-react/src/components/Tooltip/Tooltip.tokens.ts [hunk 1]
```diff
@@ -16,40 +19,37 @@ const {
...
-    placement: '-6px',
+  entities: {
+    tooltip: {
+      height: spacingXlarge,
+    },
+    arrow: {
+      width: '6px',
+      height: spacingSmall,
+      spacings: {
+        bottom: '-6px',
```
Comment (pomfrida) [inline libraries/core-react/src/components/Tooltip/Tooltip.tokens.ts:42]: Any reason you removed `spacingSmall` here?

Comment (mimarz) [reply libraries/core-react/src/components/Tooltip/Tooltip.tokens.ts:42]: Which `spacingSmall` are you referring to? I haven't removed anything I think... 🤔

Comment (pomfrida) [reply libraries/core-react/src/components/Tooltip/Tooltip.tokens.ts:42]:
```
spacings: {
    left: spacingSmall,
    right: spacingSmall,
    top: spacingSmall,
    bottom: spacingSmall,
 }
```
   But I see now these were not really removed, I was confused by the new arrow spacings 😆

**Review by pomfrida — APPROVED** [review]
LGTM 👍

**Pull request merged**